testing/resources/test_repo/broken_module/tests/test_model.py

```diff
@@ -11,3 +11,15 @@ def method1(self, example_var):
     def test_1(self):
         self.partner.message_post(body="Test", subtype="mail.mt_comment")
         self.partner.message_post("Test", subtype="mail.mt_comment")
+    
+    def test_base_method_1(self):
+        # Should fail
+        super().test_base_method_1()
+        # Should not fail
+        return super().test_base_method_2()
+
+    def test_base_method_2(self):
+        # Should not fail
+        super().test_base_method_1()
```

Comment:
Could you add cases using the class name, please? To check if those also work.

I mean:

`super(TestModel, self).test_base_method_1()`

Reply:
Added ✅ They do work, thank you!